content/news/012.dd-ix-at-ripe89.de.md

```diff
@@ -0,0 +1,16 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk und viele gute Erinnerungen
+title: DD-IX auf RIPE89 in Prag
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens und Melody von DD-IX besuchten gemeinsam die RIPE-Community bei der diesjährigen Herbst-RIPE in Prag.
```

Comment:
Marcel, Klemens und Melody nahmen am 89. Treffen der RIPE Community teil. RIPE89 fand in Prag statt.